src/data/markdown/docs/01 guides/03 Results visualization/Apache Kafka.md

```diff
@@ -3,7 +3,11 @@ title: "Apache Kafka"
 excerpt: ""
 ---
 
-You can also push the emitted metrics to [Apache Kafka](https://kafka.apache.org). You can configure the broker (or multiple ones), topic and message format directly from the command line parameter like this:
+You can also push the emitted metrics to [Apache Kafka](https://kafka.apache.org).
```

Comment:
Maybe we should elaborate on possible use cases for this, or a brief description of what Kafka is?